services/request_service.go

```diff
@@ -28,10 +31,12 @@ func NewRequestService(didMethod string, ledgerService LedgerServiceI) RequestSe
 func (rs RequestService) ResolveDIDDoc(c echo.Context) error {
```

Comment:
Am I right that this function should be covered be comments for swagger generation? Cause it handles one of routes for echo server.

Reply:
Yes, exactly!